benches/mmap/mod.rs

```diff
@@ -53,10 +53,10 @@ enum AccessKind {
 }
 
 impl AccessKind {
-    fn make_offset(&self, access_size: usize) -> u64 {
+    fn make_offset(&self, access_size: usize) -> usize {
```

Comment:
Can't we leave this line as it was before and avoid all the `let obj_off = access.make_offset(size_of::<SmallDummy>()) as u64;` changes? You are also doing now an unsafe conversion below: the `idx as usize` (idx is an u64).